### PR TITLE
fix: add new providers in readme

### PR DESCRIPTION
**Pull request by jainpawan21**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,14 +158,23 @@ Novu provides a single API to manage providers across multiple channels with a s
...
 - [ ] Bandwidth
 - [ ] RingCentral
 
 #### 📱 Push
 
 - [x] [FCM](https://github.com/novuhq/novu/tree/main/providers/fcm)
 - [x] [Expo](https://github.com/novuhq/novu/tree/main/providers/expo)
-- [x] [SNS](https://github.com/novuhq/novu/tree/main/providers/sns)
+- [x] [APNS](https://github.com/novuhq/novu/tree/main/providers/apns)
+- [ ] OneSignal
```
Comment (unicodeveloper) [inline README.md:177]: No links to Onesignal and Pushwoosh?

File: README.md [hunk 2]
```diff
@@ -156,14 +158,23 @@ Novu provides a single API to manage providers across multiple channels with a s
...
 - [x] [Gupshup](https://github.com/novuhq/novu/tree/main/providers/gupshup)
+- [x] [SMS Central](https://github.com/novuhq/novu/tree/main/providers/sms-central)
+- [x] [Maqsam](https://github.com/novuhq/novu/tree/main/providers/maqsam)
+- [x] [46elks](https://github.com/novuhq/novu/tree/main/providers/forty-six-elks)
+- [x] [Clickatell](https://github.com/novuhq/novu/tree/main/providers/clickatell)
+- [x] [Burst SMS](https://github.com/novuhq/novu/tree/main/providers/burst-sms)
+- [x] [Firetext](https://github.com/novuhq/novu/tree/main/providers/firetext)
+- [x] [Infobip](https://github.com/novuhq/novu/tree/main/providers/infobip)
+- [x] [SNS](https://github.com/novuhq/novu/tree/main/providers/sns)
 - [ ] Bandwidth
```
Comment (unicodeveloper) [inline README.md:169]: Same here. Any reason why we are not linking to it?

File: README.md (hunk 1 above)
Comment (jainpawan21) [reply README.md:177]: These providers are not yet available

File: README.md (hunk 2 above)
Comment (jainpawan21) [reply README.md:169]: These providers are not yet available

**Review by p-fernandez — APPROVED** [review]

**Pull request merged**